code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -146,7 +149,10 @@
 			return
 		else if (href_list["eject"])
 			if(beaker)
-				beaker:loc = src.loc
```

Comment:
what the fuck

Reply:
don't ask it was like that before i swear i have no idea why it's like that